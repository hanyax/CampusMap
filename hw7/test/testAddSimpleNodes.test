# Test adding only nodes to graph 

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Print the nodes in the graph 
ListNodes graph1