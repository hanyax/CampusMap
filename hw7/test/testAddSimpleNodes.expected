# Test adding only nodes to graph 

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3 n4
